java/src/jmri/jmrit/dispatcher/TrainInfoFile.java

```diff
@@ -98,6 +98,18 @@ public TrainInfo readTrainInfo(String name) throws org.jdom2.JDOMException, java
                     } else {
                         log.error("Transit name missing when reading TrainInfoFile {}", name);
                     }
+                    if (traininfo.getAttribute("dynamictransit") != null ) {
+                        tInfo.setDynamicTransit(false);
+                        if (traininfo.getAttribute("dynamictransit").getValue().equals("yes")) {
+                            tInfo.setDynamicTransit(true);
+                        }
```

Comment:
You can simplify these lines by:

setDynamicTransit(traininfo.getAttribute("dynamictransit").getValue().equals("yes"));

I might have spelling errors since I review the PR on my phone.